README.md

```diff
@@ -53,15 +53,23 @@ git clone https://github.com/flang-compiler/f18.git
 
 ### Supported C++ compilers
 
-F18 is written in C++17.
+F18 is written in C++17 and it has LLVM libraries dependency.
 
 The code has been compiled and tested with
-GCC versions 7.2.0, 7.3.0, 8.1.0, and 8.2.0.
+GCC versions from 7.2.0 to 9.3.0.
 
 The code has been compiled and tested with
-clang version 7.0 and 8.0
+clang version 7.0, 8.0, 9.0 and 10
 using either GNU's libstdc++ or LLVM's libc++.
 
+The code has been compiled on
+AArch_64, X86_64 and PPC64 servers
```

Comment:
What are the correct spellings?  Maybe aarch64, x86_64, and ppc64le, but we should model the llvm targets.  I don't know the answer.

Reply:
AArch64 is the correct spelling for that one.

I think Steve's suggestion for intel and power is fine.

Reply:
Done!